types/ssh2-sftp-client/ssh2-sftp-client-tests.ts

```diff
@@ -1,42 +1,40 @@
-import Client = require('ssh2-sftp-client');
-import * as fs from 'fs';
+import * as Client from "ssh2-sftp-client";
```

Comment:
Could you please rollback this line? Namespaces object are not callable. See [this](https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this) post for details.